DSharpPlus.Core/Entities/Optional.cs

```diff
@@ -121,6 +124,12 @@ public Optional(T value)
             _ => false
         };
 
+        /// <summary>
+        /// Returns a string that represents the current object.
+        /// </summary>
+        /// <returns>A string that represents the current object</returns>
+        public override string ToString() => HasValue ? (_value?.ToString() ?? string.Empty) : "<Empty>";
```

Comment:
I don't understand. The difference between `""` and `string.Empty` is just code style.
I only added this method to aid debugging anyways. A DebuggerDisplayAttribute would also work, but this was faster to write and likely isn't often called by code.
I can adjust both if you prefer them the other way.

Reply:
I was referencing the `<Empty>`, apologies for the lack of clarification. Now that I'm thinking about it, it shouldn't really matter since as you say: It likely isn't often called by code.